client/components/sites-dropdown/docs/example.jsx

```diff
@@ -1,18 +1,55 @@
 /**
  * External dependencies
  */
-import React from 'react';
+import React, { PureComponent } from 'react';
+import { connect } from 'react-redux';
+import find from 'lodash/find';
+import { urlToSlug } from 'lib/url';
+
 
 /**
  * Internal dependencies
  */
 import SitesDropdown from 'components/sites-dropdown';
 
-export default React.createClass( {
+class SitesDropdownExample extends PureComponent {
+	static propTypes = {
+		selectedSiteId: React.PropTypes.number
+	}
+
+	constructor( props ) {
+		super( props );
 
-	displayName: 'SitesDropdown',
+		this.state = {
+			selectedSiteId: this.props.selectedSiteId
+		};
+	}
+
+	updateSelectedSite = ( siteSlug ) => {
+		const selectedSite = find( this.props.sites, site => urlToSlug( site.URL ) === siteSlug );
+		if ( selectedSite ) {
+			this.setState( { selectedSiteId: selectedSite.ID } );
+		}
+	}
 
-	render: function() {
-		return <SitesDropdown />;
+	render(){
+		return (
+			<SitesDropdown
+				selectedSiteId={ this.state.selectedSiteId }
+				onSiteSelect={ this.updateSelectedSite } />
+		);
 	}
-} );
+}
+
+const getAllSites = ( state ) => state.sites.items;
+
+const getFirstSiteId = ( state ) => Object.keys( state.sites.items )[ 0 ];
```

Comment:
We can add these as selectors in `state/sites/selectors` if we think they'd be useful in an implementation.